virt/virtual_machines/virt-accessing-vm-consoles.adoc

```diff
@@ -50,3 +50,9 @@ You can connect to a Windows virtual machine (VM) by using the desktop viewer an
 :context: desktop-viewer
 include::modules/virt-connecting-to-vm-console-web.adoc[leveloffset=+2]
 :!desktop-viewer:
+
+[role="_additional-resources"]
+[id="additional-resources_virt-accessing-vm-consoles"]
+== Additional resources
+
+* xref:../../virt/about_virt/virt-architecture.adoc#virt-about-ssp-operator_virt-architecture[About the Scheduling, Scale, and Performance (SSP) Operator]
```

Comment:
Typically, this would follow the include for the module that refers to it, rather than being at the end of the assembly. Seems it should follow the `virt-temporary-token-VNC` module directly